records_mover/db/mysql/unload_options.py

```diff
@@ -0,0 +1,19 @@
+from typing import TypedDict, Optional, Set
+from records_mover.records import DelimitedRecordsFormat
+
+
+class MySqlUnloadOptions(TypedDict):
+    character_set: str  # TODO make this a set of valid options
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker